packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/BarChartViewer/components/Bars/Bars.js

```diff
@@ -0,0 +1,127 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled, { withTheme } from 'styled-components'
+import { Box } from 'grommet'
+import { Bar } from '@vx/shape'
+import Tippy from '@tippy.js/react'
+import { SpacedText } from '@zooniverse/react-components'
+import 'tippy.js/dist/tippy.css';
+import 'tippy.js/animations/scale.css';
+
+const StyledTippy = styled(Tippy)`
+  background-color: black !important;
+  border-radius: 0 !important;
+
+  .tippy-content {
+    padding: 0;
+  }
+
+  &[data-placement^=bottom] {
+    .tippy-arrow {
+      border-bottom-color: black !important;
+    }
+  }
+
+  &[data-placement^='top'] {
+    .tippy-arrow {
+      border-top-color: black !important;
+    }
+  }
+`
+
+export function TooltipContent ({ className, label }) {
+  return (
+    <Box
+      background={{ color: 'black', dark: true }}
+      className={className}
+      elevation='medium'
+      pad={{ horizontal: 'medium', vertical: 'xsmall' }}
+      responsive={false}
+    >
+      <SpacedText weight='bold'>{label}</SpacedText>
+    </Box>
+  )
+}
+
+const StyledTooltipContent = styled(TooltipContent)`
+  font-family: ${props => props.fontFamily};
+`
+
+function Bars (props) {
+  const {
+    data,
+    xAxisLabel,
+    xScale,
+    yAxisLabel,
+    yScale,
+    yMax,
+    theme: { global: { colors, font } }
+  } = props
+
+  return data.map((datum, index) => {
+      const { color, label, value } = datum
+      const fill = colors[color] || color || colors.brand
+      const key = `bar-${label}`
+      const barHeight = yMax - yScale(value)
+      const barWidth = xScale.bandwidth()
+      const x = xScale(label)
+      const y = yMax - barHeight
+      const alt = `${xAxisLabel} ${label}: ${yAxisLabel} ${value}`
+      return (
+        <StyledTippy
+          arrow={true}
+          animation='scale'
+          content={<StyledTooltipContent fontFamily={font.family} label={value.toString()} />}
+          key={key}
+          placement='top'
+          trigger='mouseenter focus'
+        >
+          <svg
+            tabIndex='0'
```

Comment:
Does `tabindex` on SVG work in Edge? If not, then this might need `focusable` too.

Reply:
Actually, I'm not sure why this SVG is focusable.

Reply:
I checked Edge and it's working as expected. I left this as tabbable for screen reader read out for the bar chart as a whole.

Reply:
I'm confused because the child `Bar` is what should be read out, and that does have `focusable` set.

Reply:
If this SVG should be read out, it'll need an appropriate role and a label. I think it will also mask any children from being read out.

Reply:
Ok, I wasn't sure about which would take precedence or if they would work together (somehow?). It can be removed.

Reply:
Removing the tab index from the svg element breaks the tooltips from showing at all in Firefox :(

Reply:
Ok, after reviewing the tippy.js-react docs, I remember now why I did this. It wasn't for voice over, but in fact a workaround for the fact that vx doesn't forward refs in the `Bar` component: 

> If you're using a library that doesn't forwardRef for you, and doesn't give access to the ref via innerRef or similar, you can use a wrapper <span> element as a workaround.

I used an svg element since we're within an svg. Is there a better element I can use like span but works in an svg?

Reply:
The example showed a tab index being used on the workaround wrapper element: https://github.com/atomiks/tippy.js-react#component-children

Reply:
What's confusing here is that `tabIndex` is defined on the wrapper, but `focusable` and the accessible label are added its child. They should really all be declared together, not on different elements.

Reply:
>Removing the tab index from the svg element breaks the tooltips from showing at all in Firefox :(

Did you remove it or move it to the `Bar`? It'll need to be present somewhere in order for the element to be focusable in the browser.

Reply:
Yes, I tried that first.

Reply:
Since the documentation demonstrates this is the workaround for situations where forward refs aren't correctly supported, I'd like to leave this as is. I think it'd be appropriate to open an issue on the vx repository about the forwarding refs issue, but I don't think I can drop this implementation without dropping vx and/or tippy and I don't really want to do either of those things since they generally are working. Is having an issue to track this to come back to it ok for now?

Reply:
That's fine as long as you move `focusable` and the accessible labels up from the `Bar` component to this wrapper. The change I'm asking for is that focusable behaviour is currently declared on two different elements.

>What's confusing here is that tabIndex is defined on the wrapper, but focusable and the accessible label are added to its child. They should really all be declared together, not on different elements.